### ISSUE TITLE
效果不好

### ISSUE THREAD

**Issue by luotianhang** [asyml/vision_transformer#1]
作者您好 我在cifar10 和100 上测试 效果比一般的卷积网络还要差这是为什么啊

**Comment by Hhhhhhao** [comment]
If you haven't fine-tune the network on CIFAR please do that. The pretrained model is trained using ImageNet. Our experiments show that even fine-tune the pretrained model with very small batch size we can still get similar accuracy as in the paper.

**Comment by luotianhang** [comment]
thanks for your work,  appreciate it
